Game/author.rb

```diff
@@ -0,0 +1,16 @@
+class Author
+  attr_accessor :first_name, :last_name
+  attr_reader :id, :items
+
+  def initialize(first_name, last_name)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items.push(self) unless @items.include?(item)
+    author.item = self
```

Comment:
- [ ] The Author class has no item property and I don't see any author variable declared. Please make sure this line of code is necessary.